package.json

```diff
@@ -22,7 +22,7 @@
     "lint": "lerna run lint",
     "test": "lerna run --parallel test",
     "start:ui": "lerna run --scope=@nervosnetwork/neuron-ui start",
-    "start:wallet": "lerna run --scope=@nervosnetwork/neuron-wallet start",
+    "start:wallet": "lerna run --scope=@nervosnetwork/neuron-wallet watch",
```

Comment:
What's the difference between `start` and `watch`?

Reply:
`watch` will restart `neuron-wallet` once it changed.

Reply:
I see. 👍